app/js/controllers/LoginController.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+import module from '../app';
+
+module.controller('LoginController', ($scope, LoginFactory) => {
+	$scope.user = {
+		email: '',
+		password: ''
+	};
+
+	$scope.login = () => {
+		console.log();
+
+		LoginFactory.login($scope.user).$promise.then(() => {

```

Comment:
Maybe better will be to set that expression on few lines for better readability.